### PR TITLE
[DOCS] MITRE ATT&CK being added to alert details flyout and alert reason re-named

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2686.

Preview:
- [View detection alert details](https://security-docs_2772.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html): Refreshed the `alert-details-flyout.png` screenshot.
- [View detection alert details | Overview](https://security-docs_2772.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#alert-details-overview) - Doc'd what the MITRE section was for and changed "Alert-rendered reason statement" to "Alert reason" since that update was added in 8.6 as well.

### CODE REVIEW

**Review by michaelolo24 — APPROVED** [review]
Looks great! Thanks @nastasha-solomon !

File: docs/detections/alerts-view-details.asciidoc [hunk 1]
```diff
@@ -27,7 +27,9 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Alert-rendered reason statement*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
+* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.
```
Comment (joepeeples) [inline docs/detections/alerts-view-details.asciidoc:30]: Nit: I think the full name would be best here. "MITRE" is the name of the organization that does the research, and "ATT&CK" is the name of their knowledge-base.
```suggestion
* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and subtechniques.
```

**Review by joepeeples — COMMENTED** [review]
One nit, LGTM!

File: docs/detections/alerts-view-details.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/detections/alerts-view-details.asciidoc:30]: Makes sense, I'll add this change - thanks!

File: docs/detections/alerts-view-details.asciidoc [hunk 2]
```diff
@@ -27,7 +27,9 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Alert-rendered reason statement*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
+* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and subtechniques.
+
+* *Alert reason*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```
Comment (benironside) [inline docs/detections/alerts-view-details.asciidoc:32]:
```suggestion
* *Alert reason*: Provides alert details in a logical arrangement. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```

**Review by benironside — COMMENTED** [review]
LGTM, left one minor suggestion for your consideration

**Comment by samratbhadra-qasource** [comment]
Hi @nastasha-solomon 

We have reviewed the shared preview for **MITRE ATT&CK being added to alert details flyout and alert reason re-named** and all the required changes are present and correct. ✔️

**Screenshot:**
Capture
Screenshot (1160)

Hence we marking this PR as **QA Validated**.

Thanks!

**Review by jmikell821 — COMMENTED** [review]
Very small nit, then good to go -- thanks!

File: docs/detections/alerts-view-details.asciidoc [hunk 3]
```diff
@@ -27,7 +27,9 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Alert-rendered reason statement*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
+* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and subtechniques.
+
+* *Alert reason*: Provides alert details in a logical arrangement. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```
Comment (nastasha-solomon) [inline docs/detections/alerts-view-details.asciidoc:32]:
```suggestion
* *Alert reason*: Provides alert details in a logical arrangement. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive; hover over one to access the available actions.
```

**Pull request merged**